test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +52,16 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    private void isAnyNull_noNull_False(Object... objects) {
+    	assertFalse(Utils.isAnyNull(objects));
+    }
+    
+    private void isAnyNull_oneNull_True(Object... objects) {
+    	assertTrue(Utils.isAnyNull(objects));
+    }
+    
+    private void isAnyNull_emptyList_False(Object... objects) {
```

Comment:
As mentioned in tutorial, there is no need to abstract out `assertTrue` and `assertFalse` as separate methods since they are rather short. Existing tests do that to avoid repeating `Arrays.asList` everywhere. You can use them directly in `isAnyNull`.